cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,16 @@
+/// <reference types="cypress" />
+
+import { generateUser } from '../support/generate';
+
 describe('Sign Up page', () => {
   it('should provide an ability to register new account', () => {
+    const user = generateUser();
 
+    cy.visit('user/register');
+    cy.get('input[placeholder="Username"]').type(user.username);
+    cy.get('input[type="email"]').type(user.email);
+    cy.get('input[type="password"]').type(user.password);
+    cy.get('button[type="submit"]').click();
```

Comment:
After clicking the submit button, it is a good practice to check if the form was submitted successfully. You can add a check to see if the page has been redirected to the expected URL.